README.md

```diff
@@ -9,15 +9,41 @@ It can take advantage of various transport interfaces provided by Janus API, mor
 This package is experimental and is not yet released to hex.
 
 ## Example
+```elixir
+# handler example
+defmodule CustomHandler do
+  use Janus.Handler
+
+  @impl true
+  def init(_) do
+    {:ok, %{}}
+  end
+  
+  # example of event's callback implementation
+  @impl true
+  def handle_created(_session_id, _transport, _emitter, _timestamp, state) do
+    # created event has been send by the gateway, handle it in any way e.g. log, store in database
+    {:noreply, state}
+  end
+  
+  ...
+  
+end
+```
+
+Communicating with the gateway:
 
 ```elixir
 # user have to provide transport and handler modules that ElixirJanus can take advantage of
+# this example uses previously created `CustomHandler` and `Janus.Transport.WS` package for transport
 iex> alias Janus.{Connection, Session}
-iex> {:ok, connection} = Connection.start_link(your_transport_module, transport_args, your_handler_module, handler_args, []) 
+iex> alias Janus.Transport.WS
+iex> {:ok, connection} = Connection.start_link(WS, {"WebSocket url to the gateway", WS.Adapters.WebSockex, [timeout: 5000]}, CustomHandler, {}, []) 
```

Comment:
I have one more idea (although not for this PR of course) - We cannot use WebSockex adapter with any other transport, so the adapter could define a callback providing transport module and get rid of the one argument from here.
We might also consider setting adapter via config - I don't think anyone will ever need to use 2 adapters in one app